Rubberduck.VBEEditor/SafeComWrappers/Abstract/IDesigner.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+using System.Collections;
+
+namespace Rubberduck.VBEditor.SafeComWrappers.Abstract
+{
+    public interface IDesigner : ISafeComWrapper, IEquatable<IVBComponent>
+    { 
+        IList Selected { get; }
```

Comment:
Confused. Why was `IVBComponent.SelectedControls` not used instead?

Reply:
Needed to accommodate the late-binding dynamic nature of .Designer (COM Object) - only used for mocks atm, not in the live code.

Reply:
I know, but `IVBComponent.SelectedControls` *already* gives you the selected controls given the `VBComponent` for the userform... or did I miss something?

Reply:
It's *far* more likely that I missed something ;-)

So the challenge I had was when testing this code in FindFormDesignerTarget():

var designer = ((dynamic)component.Target).Designer; // COM Object, need to late bind
int selectedCount = designer.**Selected**.Count;

Works fine in live, but for the mock I needed Designer to return an object that has a Selected Property with a Count off of it.

Is there a better way I've missed?

Reply:
Sorry, just grokked it. Working on it...